admin/client.py

```diff
@@ -44,7 +41,7 @@ class DockerImage(object):
 }
 
 PIP_DISTRIBUTIONS = DOCKER_IMAGES.keys()
-PACKAGED_CLIENT_DISTRIBUTIONS = ('ubuntu-14.04', 'ubuntu-15.04')
+PACKAGED_CLIENT_DISTRIBUTIONS = ('centos-7', 'ubuntu-14.04', 'ubuntu-15.04')
```

Comment:
Add a comment explaining what this means. I guess it means "Linux distributions which we create CLI packages for". 

This change implies that we support the CentOS 7 packages, but installing them is not described on http://doc-dev.clusterhq.com/install/install-client.html so I think that this isn't a needed change. Or at least, it is a separate change.